Updates for TPM 3.0 and above (#8822)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineDialInConferencingUser.md

The see a list of users with conferencing providers other than Microsoft use the Get-CsUserAcp cmdlet.

**NOTE:**

> Beginning Teams PowerShell Module 3.0 onwards, the below updates are applicable for TeamsOnly customers.

Reviewer: ```suggestion
Beginning with Teams PowerShell Module 3.0 onwards, the following updates are applicable for TeamsOnly customers.
```

*Deprecated Input Parameters*

> Below input paramters have been removed as they are either no longer relevant in Microsoft Teams or have very low usage

Reviewer: ```suggestion
The following input parameters have been removed as they are either no longer relevant in Microsoft Teams or have very low usage.
```

  - ServiceNumber
  - BridgeId
  - BridgeName
  - LdapFilter
  - DomainController
  - Force
  - TenantDomain
  - Common Parameters

## PARAMETERS

### -BridgeId

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

The see a list of users with conferencing providers other than Microsoft use the Get-CsUserAcp cmdlet.

**NOTE:**

Beginning with Teams PowerShell Module 3.0 onwards, the following updates are applicable for TeamsOnly customers.

*Deprecated Input Parameters*

The following input parameters have been removed as they are either no longer relevant in Microsoft Teams or have very low usage.

  - ServiceNumber
  - BridgeId
  - BridgeName
  - LdapFilter
  - DomainController
  - Force
  - TenantDomain
  - Common Parameters

## EXAMPLES

### -------------------------- Example 1 --------------------------
```
PS C:\> Get-CsOnlineDailinConferncingUser -Identity Ken.Myer@contoso.com
```

This example uses the User Principal Name (UPN) to retrieve the BridgeID and ServiceNumber information.

## PARAMETERS

### -BridgeId

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -BridgeName

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -BridgeName

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -ServiceNumber

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -ServiceNumber

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -DomainController

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -DomainController

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -Force

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -Force

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -LdapFilter

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -LdapFilter

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -Tenant

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -Tenant

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -TenantDomain

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

```

### -TenantDomain

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### CommonParameters

> *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
